path_oidc.go

```diff
@@ -46,12 +53,25 @@ func pathOIDC(b *jwtAuthBackend) []*framework.Path {
 				"code": {
 					Type: framework.TypeString,
 				},
+				"id_token": {
+					Type: framework.TypeString,
+				},
 			},
 
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ReadOperation: &framework.PathOperation{
 					Callback: b.pathCallback,
 					Summary:  "Callback endpoint to complete an OIDC login.",
+
+					// state is cached so don't process OIDC logins on perf standbys
+					ForwardPerformanceStandby: true,
```

Comment:
This is my first time working with this field and it's not deeply described on the path operation. The way I read it is, by default it's false, so by default it would read "ForwardPerformanceStandby: false" and the resulting behavior would be that we _would not_ forward requests to performance standby's, we'd only handle them on the leader. 

So the way I read this, setting it to true would cause it to yes, forward requests to performance standbys. However, the comment reads the opposite way.

Am I thinking of this incorrectly? Same question for line 74.

Reply:
Nevermind! I had a chance to go read the Vault code and answered my own question.